test_unstructured_ingest/check-diff-evaluation-metrics.sh

```diff
@@ -13,43 +13,50 @@ set -e
 SCRIPT_DIR=$(dirname "$(realpath "$0")")
 OVERWRITE_FIXTURES=${OVERWRITE_FIXTURES:-false}
 TMP_DIRECTORY_CLEANUP=${TMP_DIRECTORY_CLEANUP:-true}
+OUTPUT_FOLDER_NAME=$1
```

Comment:
this env var is confusing. call this `EVAL_NAME` to be consistent with evaluation-metrics.sh?